.github/drafts/PULL_REQUEST_TEMPLATE/new_feature.md

```diff
@@ -7,11 +7,19 @@ assignees: ''
 
 ---
 
-## This PR adds...
+## Double check these details before you open a PR**
+*Tick the checkboxes to ensure you've done everything*
+- [] PR does not match another non-stale PR currently opened
+- [] PR name matches the format *New Feature: brief description of feature*
+
+
+## This PR adds/fixes...
 
 *List your features here and the benefits they bring.*
 
 ## Notes
 
 *List anything note-worthy here (potential issues, this needs to be merged to `master` before working, etc....).*
 *Don't forget to link any issues that this PR will solved.*
+
+**Please link any issue that this PR should solve if it's applicable**
```

Comment:
```suggestion
<!-- List anything note-worthy here (potential issues, this needs to be merged to `master` before working, etc....).
Don't forget to link any issues that this PR will solve.

Please link any issue that this PR solves if applicable -->
```

Reply:
Same here for the comments. The verb tenses fixes are appreciated.